model/src/main/proto/question.proto

```diff
@@ -81,3 +81,17 @@ message AnsweredQuestionOutcome {
   // Whether the answer the learner submitted is the correct answer.
   bool is_correct_answer = 2;
 }
+
+// The final score and mastery calculations of the user after completing a training session.
```

Comment:
Is this an ephemeral data structure or something that will be persisted on disk? That should be clarified in the documentation to better set expectations. One has very different maintenance responsibilities than the other (e.g. persisted protos need to be carefully updated to avoid breaking backward compatibility).